lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -2888,6 +2888,109 @@ bool LowererMDArch::GenerateFastAnd(IR::Instr * instrAnd)
     return true;
 }
 
+bool LowererMDArch::GenerateFastDiv(IR::Instr * instrDiv)
+{
+    Assert(instrDiv);
+    auto * divisor = instrDiv->GetSrc2(); // denominator
+
+    if (PHASE_OFF(Js::BitopsFastPathPhase, this->m_func) || !divisor->IsIntConstOpnd() ||
+        !(instrDiv->m_opcode == Js::OpCode::Div_I4 || instrDiv->m_opcode == Js::OpCode::Rem_I4)) //ToDo Optimize unsigned division
+    {
+        return false;
+    }
+
+    auto * divident = instrDiv->GetSrc1(); // nominator
+    auto * dst = instrDiv->GetDst();
+    auto constDivisor = divisor->AsIntConstOpnd()->AsInt32();
+    auto isNegDevisor = false;
```

Comment:
>auto [](start = 4, length = 4)

We avoid auto in general except for large unreadable template names.  Could we avoid it in this function?  It makes the code less readable.

Reply:
sure. fixed.